src/controllers/members.js

```diff
@@ -0,0 +1,74 @@
+const Member = require('../models/Member');
+
+const regexObjectId = /^[0-9a-fA-F]{24}$/;
+
+const getMemberById = (req, res) => {
+  const { id } = req.params;
+  if (!id.match(regexObjectId)) {
+    res.status(400).json({
```

Comment:
you should return this response, because we do not want execute rest of the endpoint